topics/awesome/index.md

```diff
@@ -0,0 +1,12 @@
+---
+display_name: Awesome List
```

Comment:
Think this should be `Awesome Lists`, right?

Reply:
Yeah, I somehow just transformed `awesome-list` into `Awesome List` without thinking too much about it. In case of other topics like `nodejs` that would have worked, but here `Awesome Lists` is 
actually more appropriate.